massa-async-pool/src/pool.rs

```diff
@@ -21,8 +21,8 @@ pub struct AsyncPool {
     /// async pool config
     config: AsyncPoolConfig,
 
-    /// messages sorted by increasing ID (increasing priority)
-    messages: BTreeMap<AsyncMessageId, AsyncMessage>,
+    /// messages sorted by decreasing ID (decreasing priority)
```

Comment:
Not sure I see how the id is related to the priority

Reply:
we uniquely identify every message by its priority tuple :) that way we have only one type of ID to use in the BTreeMap

Reply:
It feels odd calling that an id as everywhere else ids are hashes, but maybe I'm getting too mush used to this code base specificity :sauropod: